core-job/src/main/java/org/apache/kylin/job/dao/ExecutableDao.java

```diff
@@ -393,7 +395,14 @@ public ExecutableOutputPO getJobOutput(String uuid) throws PersistentException {
             return result;
         } catch (IOException e) {
             logger.error("error get job output id:" + uuid, e);
-            throw new PersistentException(e);
+            if(e.getCause() instanceof FileNotFoundException) {
+                result = new ExecutableOutputPO();
+                result.setUuid(uuid);
+                result.setStatus(ExecutableState.SUCCEED.name());
+                return result;
+            }else {
```

Comment:
This seems not formatted; Please double check and ensure your IDE has been configured.

Reply:
OK, I have fixed it.